### PR TITLE
Admin UI tweaks

### PR DESCRIPTION
**Pull request by jensens**
- white textcolor on button
- place link to site

### CODE REVIEW

**Comment by jensens** [comment]
@jenkins-plone-org please run jobs

**Comment by stevepiercy** [comment]
I don't suppose you have any before and after screenshots?

**Comment by jensens** [comment]
No screenshots, sorry.
a) The link on the button was was in dirty yellow and now its white.
b) I placed a link to the site after migration is done. There was not any link available and one had to edit the address bar to get anywhere.

**Comment by stevepiercy** [comment]
I'd like to offer constructive feedback, but I have not looked into building, previewing, and testing this repo. I trust that you know what you're doing, and the changes look fine to me.

**Comment by mauritsvanrees** [comment]
In the Zope root, before:

Screenshot 2022-09-13 at 17 01 24

and after:

Screenshot 2022-09-13 at 17 01 50

Much better.

And for the other part: I have been missing that link! Thanks for putting it back. I would do it in a different part of the template though. See inline comments.

**Review by mauritsvanrees — CHANGES_REQUESTED** [review]

File: Products/CMFPlone/browser/templates/plone-upgrade.pt [hunk 1]
```diff
@@ -33,7 +33,7 @@
             <h1>
               <span i18n:translate="">Upgrade this site</span>
             </h1>
-            <h2>${context/Title} <small>(${context/getId})</small></h2>
+            <h2>${context/Title} <small>(id=${context/getId})</small></h2>
```
[the remark this answers is not in the document]
Comment (jensens) [reply Products/CMFPlone/browser/templates/plone-upgrade.pt:36]: I thought about that, but having a h2 a link feels odd somehow.

File: Products/CMFPlone/browser/templates/plone-upgrade.pt [hunk 2]
```diff
@@ -45,7 +45,7 @@
         <article class="row mb-4">
           <div class="col-md-12 mb-3">
             <p class="alert-success p-2" tal:condition="versions/equal">
-              <span i18n:translate="" tal:omit-tag="">Your site is up to date.</span>
+              <span i18n:translate="" tal:omit-tag="">Your <a href="${context/absolute_url}">site</a> is up to date.</span>
```
[the remark this answers is not in the document]
Comment (jensens) [reply Products/CMFPlone/browser/templates/plone-upgrade.pt:48]: ok, this is a reason for my lazy me to ignore its a h2....

**Comment by jensens** [comment]
I really do not like having a fat link on a h2. so, my idea is to place the id as the path (which is IMO the more valuable information) linked to the site below the title.

**Comment by jensens** [comment]
@stevepiercy sorry, next time I will provide screenshots. It's just - my i3wm based setup lost the capability to take screenshots. Now I am able again to provide those.

**Comment by stevepiercy** [comment]
The last screenshot looks good to me!

**Review by mauritsvanrees — APPROVED** [review]

**Pull request merged**